draft-ietf-rats-corim.md

```diff
@@ -1216,15 +1216,19 @@ matched.
 {::include cddl/stateful-environment-record.cddl}
 ~~~
 
-The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
-Each `conditional-series-record` record is evaluated in the order it appears in the series array.
-The Endorsed Values are accepted if the series condition in a `conditional-series-record` matches the ACS.
-The first `conditional-series-record` that successfully matches an ACS Entry terminates the matching and the corresponding Endorsed Values are accepted.
-If none of the series conditions match an ACS Entry, the triple is not matched,
-and no Endorsed values are accepted.
+The series is an array of `conditional-series-record` that has a `selection` and an `addition`, both expressed as a list of `measurement-map`.
+The `selection` and `addition` operate within the scope of the `conditional-endorsement-series-triple-record`'s matching `condition`.
 
-The `authorized-by` value in `measurement-map` in the stateful environment, if present,
-applies to all measurements in the triple, including `conditional-series-record` records.
+For each `conditional-series-record` entry, if the `selection` matches the ACS entry, the `addition` is added to the ACS.
+
+The first `conditional-series-record` entry that successfully matches the ACS entry terminates the series.
+For a `conditional-series-record` to match, every measurement in the `measurement-map` list MUST match a measurement in the ACS entry.
+
+If none of the `selection` values match in the ACS entry, the triple is not matched, and no `addition` values are accepted.
+
+If the `authorized-by` value is present in the triple `condition` (i.e., in the `measurement-map` of the `stateful-environment-record`), all authority entries of the `condition` MUST be present in the ACS entry, otherwise the ACS entry does not match.
```

Comment:
Is this needed here>
Should we be discussing how to transform external encoding into ECT encoding so that the text in 8.9 (which already says this) can be used?

Reply:
> Is this needed here> Should we be discussing how to transform external encoding into ECT encoding so that the text in 8.9 (which already says this) can be used?

I think it's OK to describe the semantics of the triple in terms of what to expect will be reflected in the ACS. The transformation section describes how to map from external to internal. The Phase 4 processing section describes how the ACS realizes an ending state. This section describes what to expect in terms of the ending state in ACS - i.e., it's a what vs. how difference.